data/pathfinder/paizo/roleplaying_game/bestiary/b1_abilities_race.lst

```diff
@@ -17,7 +17,7 @@ Racial Traits ~ Dragon (Blue)		CATEGORY:Internal
 Racial Traits ~ Dragon (Green)		CATEGORY:Internal
 Racial Traits ~ Dragon (Red)		CATEGORY:Internal
 Racial Traits ~ Dragon (White)		CATEGORY:Internal
-Racial Traits ~ Elemental (Air)		CATEGORY:Internal
+#Racial Traits ~ Elemental (Air)		CATEGORY:Internal
```

Comment:
Why is this commented out?

Reply:
I originally was going to comment out all of the elemental traits abilities, since they're now also in ce_abilities_familiar_race_cr.lst, but then I didn't go back and to the others after I finished testing the first one. I'll uncomment it, but it doesn't matter either way for functionality.